### PR TITLE
[Customers Product]: Change references from 'Users' to 'Customers' within documentation 👥

### PR DESCRIPTION
**Pull request by sumitramanga**
## [Customers Product]: Change references from 'User tracking' to 'Customers' 👤

**NOTE: This is not to be merged until we have fully released the name change in the Raygun application. Changes to 'users' within the code needs to be discussed and planned for.**

### Description 📝 
This PR is to updating the name convention for 'User tracking' to now be 'Customers' to reflect what's within the Raygun application.

**Updates**
👉 Update `README` documentation
👉 Update `CHANGELOG` documentation

### Test plan 🧪 
- Make sure documentation changes are reflected to be 'Customers' instead of 'User Tracking'

### Checklist ✔️ 
- [ ] Builds pass
- [ ] Reviewed by another developer
- [ ] Code is written to standards

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Bugfixes
 ## 2.4.0 (31/07/2017)
 
 Features
-  - Add functionality to track affected user in Sidekiq jobs, refer to the README for more information, under the "Affected User Tracking in Sidekiq" heading ([#121](https://github.com/MindscapeHQ/raygun4ruby/pull/121))
+  - Add functionality to track affected user/customer in Sidekiq jobs, refer to the README for more information, under the "Affected User Tracking/Customers in Sidekiq" heading ([#121](https://github.com/MindscapeHQ/raygun4ruby/pull/121))
```
Comment (Deacon-McIntyre) [inline CHANGELOG.md:62]: Wondering why the changes in this file use the slash instead of replacing the "affected user" text?

Comment (sumitramanga) [reply CHANGELOG.md:62]: As this is a changelog I thought it would be best to not remove content from here. Instead add more to

Comment (Deacon-McIntyre) [reply CHANGELOG.md:62]: Cool, makes sense to me 😃

**Review by Deacon-McIntyre — APPROVED** [review]
